addons/hr_holidays/tests/test_holidays_flow.py

```diff
@@ -146,32 +148,33 @@ def _check_holidays_status(holiday_status, ml, lt, rl, vrl):
         _check_holidays_status(hol_status_2_employee_group, 2.0, 0.0, 2.0, 1.0)
 
         # HrUser validates the first step
-        hol2_user_group.signal_workflow('validate')
+        hol2_user_group.action_approve()
         self.assertEqual(hol2.state, 'validate1',
                          'hr_holidays: first validation should lead to validate1 state')
 
-        # HrUser validates the second step
-        hol2_user_group.signal_workflow('second_validate')
+        # HrManager validates the second step
+        hol2_user_group.sudo(self.user_hrmanager_id).action_validate()
```

Comment:
Yes, button was only visible to hr_managers but anyone could validate it.
Now it's being checked, so I had to correct it in the test.